src/solver/infeasible-problem-analysis/report.cpp

```diff
@@ -24,8 +24,6 @@
 #include <regex>
 #include <typeindex>
 
-#include <antares/logs/logs.h>
-
 namespace Antares::Optimization
 {
 InfeasibleProblemReport::InfeasibleProblemReport(
```

Comment:
class **InfeasibleProblemReport** : 
Actually, what it does is : build a report for slack variable analysis.
So a better name could be : **SlackAnalysisReportBuilder**.
The associated file name could change accordingly.